### DISCUSSION TITLE
"Continue" plugin (AI code assistant) for neovim

### DISCUSSION THREAD

**Discussion by antonkratz** [neovim/neovim#28162] [General]
I have been using vim for decades, but have not used neovim yet. Recently I had good experiences with VSCode + Continue + ollama serving deepseek and starcoder. I am wondering, is it possible to have a similar setup with neovim, has someone already written the functionality that corresponds to the Continue plug in? What is "the" plug in that everyone uses for this? I found David-Kunz / gen.nvim is that the state-of-the-art? I am interesting in tab-autocompletion, infilling, but also highlighting and discussing.

**Comment by justinmk** [comment]
Just like LLMs, humans sometimes need context. I assume you mean this plugin?: https://github.com/continuedev/continue

**Reply by antonkratz** [reply]
Yes, this is the plug-in I mean.
I seek s/th like this for vim or neovim.
Intended for coding… with tab autocompletion and/or infilling. Basically the continue plugin + ollama is awesome but something like continue doesn’t seem to exist for neovim or vim… or maybe I just can’t find it. The plug-ins I found don’t seem to support autocompletion at all.

**Reply by jonahfang** [reply]
Try "supermaven-inc/supermaven-nvim"!

**Comment by meicale** [comment]
https://github.com/huggingface/llm.nvim

**Comment by lingster** [comment]
how about this: https://github.com/yetone/avante.nvim

**Comment by iturgeon** [comment]
@antonkratz looks like there's a draft pull request in the works https://github.com/continuedev/continue/pull/2098

**Comment by PLAZMAMA** [comment]
Would also highly recommend [codecompanion.nvim](https://github.com/olimorris/codecompanion.nvim) I find it easier to configure, customize and use than avante.
